src/lib/pseudo.js

```diff
@@ -1,27 +1,45 @@
 'use strict';
 
-// Recursively walk an AST node searching for content leaves
-export function walkContent(node, fn) {
-  if (typeof node === 'string') {
-    return fn(node);
+// Walk an entry node searching for content leaves
+export function walkEntry(entry, fn) {
```

Comment:
OK, I see you've thought about this and I don't mind the change.  I'm just not sure if I understand why it's a problem that the shape is the same.